android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlService.java

```diff
@@ -16,7 +16,9 @@
 import com.smartdevicelink.managers.SdlManager;
 import com.smartdevicelink.managers.SdlManagerListener;
 import com.smartdevicelink.managers.file.filetypes.SdlArtwork;
+import com.smartdevicelink.managers.lifecycle.ISessionListener;
```

Comment:
Please remove changes to this file as they relate to hello sdl

Reply:
@JulianKast Done. (Please, see c35b6f5)